hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpc/nio/NioAsyncSocket_LargePayloadTest_Nightly.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.tpc.nio;
+
+import com.hazelcast.test.annotation.SlowTest;
+import org.junit.experimental.categories.Category;
+
+@Category(SlowTest.class)
+public class NioAsyncSocket_LargePayloadTest_Nightly extends NioAsyncSocket_LargePayloadTest{
+
+    public NioAsyncSocket_LargePayloadTest_Nightly(){
+        iterations = 20000;
```

Comment:
nit: I think it would be better to set the argument via constructor

Reply:
In theory yes; in practice no. JUnit doesn't like it when your class has a constructor with arguments.